README.md

```diff
@@ -1,212 +1,41 @@
-juju
-====
+[![Juju logo](doc/juju-logo.png?raw=true)](https://jujucharms.com/)
 
-juju is devops distilled.
+## Why Juju?
 
-Juju enables you to use [Charms](https://docs.jujucharms.com/applications-and-charms) to deploy your
-application architectures to EC2, OpenStack, Azure, GCE, your data center, and
-even your own Ubuntu based laptop.  Moving between models is simple giving you
-the flexibility to switch hosts whenever you want — for free.
+* Reduce complexity  
+* Enable repeatability  
+* Codify operations knowledge  
+* Simplify day two 
+* Maintain portability
```

Comment:
I suspect this'll be confusing to folks not familiar with the idea of substrates.  'Maintain portability' makes me think of Java's "write once, run anywhere" stuff.  Perhaps "Run your applications on anything from bare metal, private cloud to any public cloud" or something?

Reply:
Yes, you're probably right here. Portability isn't portability in the CPU architecture sense. It would also be helpful to have a few keywords to aid searching. 

In an attempt to stick to a 2 or 3 word budget, what about "Cloud agnostic"?

We're also considering having a deep link to a fuller explanation of what we mean by each of the bullets.